lib/android.js

```diff
@@ -586,14 +586,17 @@ function tryDetectInstrumentationOffset () {
 }
 
 function parsex86InstrumentationOffset (insn) {
-  if (insn.mnemonic !== 'mov') {
+  if (insn.mnemonic !== 'mov' && insn.mnemonic !== 'add') {
```

Comment:
```suggestion
  const { mnemonic } = insn;
  if (mnemonic !== 'mov' && mnemonic !== 'add') {
```

(Untested. Hopefully these changes will be approved by `./node_modules/.bin/semistandard`.)

Reply:
👍